Childrens-Social-Care-CPD/Views/Shared/_AssetDownload.cshtml

```diff
@@ -0,0 +1,155 @@
+            "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
+        };
+
+        if (wordContentTypes.Contains(contentType)) return true;
+
+        return MatchFileExtension(new List<string>{"doc", "docx"}, fileName);
+    }
+
+    private bool MatchFileExtension (List<string> extensions, string fileName) {
+        bool match = false;
+        extensions.ForEach(extension => {
+            if (Regex.IsMatch(fileName,@"\." + extension + "$", RegexOptions.None, TimeSpan.FromMilliseconds(100))) match = true;
+        });
+        return match;
+    }
+}
+
+@{
+    bool fileIsOds = FileIsODS(Model.Asset.File.FileName, Model.Asset.File.ContentType);
+}
+
+<section class="gem-c-attachment govuk-!-margin-bottom-6" data-module="ga4-link-tracker" data-ga4-link-tracker-module-started="true">
+    <div class="gem-c-attachment__thumbnail govuk-!-display-none-print">
+        <a class="govuk-link" target="_self" tabindex="-1" aria-hidden="true" data-ga4-link="{&quot;event_name&quot;:&quot;file_download&quot;,&quot;type&quot;:&quot;attachment&quot;}" href="@Model.Asset.File.Url">
+            @Html.Raw(@FileTypeIconSVG(@Model.Asset.File.FileName, @Model.Asset.File.ContentType))
+        </a>
+    </div>
+    <div class="gem-c-attachment__details">
+        <h3 class="gem-c-attachment__title">
+            <a class="govuk-link gem-c-attachment__link" target="_self" data-ga4-link="{&quot;event_name&quot;:&quot;file_download&quot;,&quot;type&quot;:&quot;attachment&quot;}" href="@Model.Asset.File.Url">
+                @Model.LinkText
+            </a>
+        </h3>
+        <p class="gem-c-attachment__metadata">
+            <span class="gem-c-attachment__attribute">@Html.Raw(@FileTypeToHumanReadable(@Model.Asset.File.FileName, @Model.Asset.File.ContentType))</span>, <span class="gem-c-attachment__attribute">@FileSizeToHumanReadable(@Model.Asset.File.Details.Size)</span>
+        </p>
+
+        @if (fileIsOds)
+        {
+        <p class="gem-c-attachment__metadata">
+            This file is in an <a href="https://www.gov.uk/guidance/using-open-document-formats-odf-in-your-organisation" target="_blank" rel="noopener" class="govuk-link">OpenDocument</a> format
+        </p>    
+        }
+
+        <p class="gem-c-attachment__metadata">This file may not be suitable for users of assistive technology.</p>
+        <details data-module="ga4-event-tracker" data-ga4-event="{&quot;event_name&quot;:&quot;select_content&quot;,&quot;type&quot;:&quot;detail&quot;,&quot;text&quot;:&quot;Request an accessible format.&quot;,&quot;section&quot;:&quot;Request an accessible format.&quot;,&quot;index_section&quot;:1,&quot;index_section_count&quot;:1}" class="gem-c-details govuk-details govuk-!-margin-bottom-3" data-ga4-event-tracker-module-started="true">
+            <summary class="govuk-details__summary">
+                <span class="govuk-details__summary-text" data-ga4-expandable="">Request an accessible format.</span>
```

Comment:
Remove `.` from the string in line 148 so it reflects designs.

Reply:
Done :)